.github/workflows/go.yaml

```diff
@@ -12,6 +12,12 @@ on:
       version-is-repo-ref:
         required: true
         type: boolean
+      sdk-features-repo-path:
```

Comment:
These vars need to be set to be used below, correct? (same for other ones of these that aren't using these vars)

Reply:
Derp. Thanks.